cmake/SetupCompilers.cmake

```diff
@@ -53,6 +53,10 @@ elseif (CMAKE_CXX_COMPILER_ID MATCHES GNU)
 elseif (CMAKE_CXX_COMPILER_ID MATCHES Intel)
   set(RAJA_COMPILER "RAJA_COMPILER_ICC")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+elseif(CMAKE_CXX_COMPILER_ID MATCHES XL)
+  set(RAJA_COMPILER "RAJA_COMPILER_XLC12")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O5")
```

Comment:
Since `-O5` is not a required flag, I think that this line is unnecessary. Stuff like this can be passed to CMake in the standard way: `cmake -DCMAKE_CXX_FLAGS="-O5"`